cmd/main.go

```diff
@@ -98,6 +105,21 @@ func main() {
 		os.Exit(1)
 	}
 
+	if err = (&controller.HealthConvertReconciler{
+		Client: mgr.GetClient(),
+		Scheme: mgr.GetScheme(),
+		Agent:  a,
+	}).SetupWithManager(mgr); err != nil {
+		setupLog.Error(err, "unable to create controller", "controller", "HealthConvert")
+		os.Exit(1)
```

Comment:
this is a bit different practice, but for now I think we shouldn't exit 1 here as there could be annoying races with the crds being created that we don't anticipate and it's not good for the agent to go down entirely.

If we could drop an error log instead that'd be safer for sure.

Reply:
done